<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600000</width>
    <height>1600000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Login">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Login</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="MyResources.qrc">:/Downloads/supermarket-groceries-in-the-basket.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="LE_username">
               <property name="placeholderText">
                <string>User Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LE_password">
               <property name="placeholderText">
                <string>Password</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="pb_login">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pb_signup">
             <property name="text">
              <string>Sign up</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Admin">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>401</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Add Item</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="LE_cat">
           <property name="placeholderText">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="LE_name">
             <property name="placeholderText">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_id">
             <property name="placeholderText">
              <string>ID</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="LE_price">
             <property name="placeholderText">
              <string>Price</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_quantity">
             <property name="placeholderText">
              <string>Quantity</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pb_save_item">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>300</y>
        <width>401</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Delete item</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="LE_nameD">
             <property name="placeholderText">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_idD">
             <property name="placeholderText">
              <string>ID</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>381</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Display</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTableWidget" name="table_admin"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="pb_display">
             <property name="text">
              <string>Display</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_clear">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pb_back_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>470</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="User">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTabWidget" name="tabWidge">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Fruits</string>
          </attribute>
          <widget class="QSplitter" name="splitter_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Products</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_fruits"/>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string>Cart</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QTableWidget" name="tableCart1"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Vegitables</string>
          </attribute>
          <widget class="QSplitter" name="splitter_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_14">
            <property name="title">
             <string>Products</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_16">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Vegi"/>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_15">
            <property name="title">
             <string>Cart</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_17">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Cart2"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Canned Foods</string>
          </attribute>
          <widget class="QSplitter" name="splitter_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_16">
            <property name="title">
             <string>Products</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_18">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Canned"/>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_17">
            <property name="title">
             <string>Cart</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_19">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Cart3"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Daity</string>
          </attribute>
          <widget class="QSplitter" name="splitter_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_18">
            <property name="title">
             <string>Products</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_20">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Dairy"/>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_19">
            <property name="title">
             <string>Cart</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_21">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Cart4"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Cold Cuts</string>
          </attribute>
          <widget class="QSplitter" name="splitter_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_20">
            <property name="title">
             <string>Products</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_22">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Cold"/>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_21">
            <property name="title">
             <string>Cart</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_23">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Cart5"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>Clothes</string>
          </attribute>
          <widget class="QSplitter" name="splitter_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_22">
            <property name="title">
             <string>Products</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_24">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Clothes"/>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_23">
            <property name="title">
             <string>Cart</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_25">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Cart6"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Snakes</string>
          </attribute>
          <widget class="QSplitter" name="splitter_11">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox_24">
            <property name="title">
             <string>Products</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_26">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Snacks"/>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_25">
            <property name="title">
             <string>Cart</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_27">
             <item row="0" column="0">
              <widget class="QTableWidget" name="table_Cart7"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_10">
          <attribute name="title">
           <string>Cleaning</string>
          </attribute>
          <widget class="QSplitter" name="splitter_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>741</width>
             <height>441</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QGroupBox" name="groupBox_26">
             <property name="title">
              <string>Products</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_28">
              <item row="0" column="0">
               <widget class="QTableWidget" name="table_Cleaning"/>
              </item>
             </layout>
            </widget>
            <widget class="QGroupBox" name="groupBox_27">
             <property name="title">
              <string>Cart</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_29">
              <item row="0" column="0">
               <widget class="QTableWidget" name="table_Cart8"/>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_back">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_checkout">
            <property name="text">
             <string>Checkout</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Checkout">
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>368</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>Cart</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTableWidget" name="table_Cart9"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>371</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Address</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="LE_email">
             <property name="placeholderText">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_number">
             <property name="placeholderText">
              <string>Phone Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_city">
             <property name="placeholderText">
              <string>City</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_gov">
             <property name="placeholderText">
              <string>Governorate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_dis">
             <property name="placeholderText">
              <string>District</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_building">
             <property name="placeholderText">
              <string>Building no.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LE_apt">
             <property name="placeholderText">
              <string>Apt. number</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pb_order">
             <property name="text">
              <string>Order</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_back_2">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_save">
             <property name="text">
              <string>Save Address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_load">
             <property name="text">
              <string>Load Address</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionada">
   <property name="text">
    <string>ada</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>table_admin</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
